deps.bzl

```diff
@@ -308,6 +308,7 @@ genrule(
   cmd = '''
     DIR=$$(dirname $(execpath _config.yml))
     $(execpath @jekyll_nix//:bin/jekyll) build -s $$DIR
+    rm _site/BUILD.bazel _site/LICENSE _site/README.md _site/WORKSPACE
```

Comment:
Maybe add those to `exclude` in `glob` above instead?

Reply:
moved to `exlude`